src/shaders/depth.frag

```diff
@@ -6,20 +6,35 @@ in highp vec2 textureCoordinates;
 #else
 in highp float depth;
 #endif
-
+#ifdef DEPTH_VISUALIZATER
+uniform highp float depthScaling;
+out highp vec4 fragmentColor;
+#else
 out highp float originalDepth;
+#endif
 
 void main() {
   #ifdef UNPROJECT_EXISTING_DEPTH
-  highp float depth = texture(depthTexture, textureCoordinates).r;
-  originalDepth =
-    #ifndef NO_FAR_PLANE_PATCHING
-    /* We can afford using == for comparison as 1.0f has an exact
-       representation and the depth is cleared to exactly this value. */
-    depth == 1.0 ? 0.0 :
+    highp float depth = texture(depthTexture, textureCoordinates).r;
+    #ifdef DEPTH_VISUALIZATER
+    highp float
     #endif
-    depthUnprojection[1] / (depth + depthUnprojection[0]);
+    originalDepth =
+      #ifndef NO_FAR_PLANE_PATCHING
+      /* We can afford using == for comparison as 1.0f has an exact
+         representation and the depth is cleared to exactly this value. */
+      depth == 1.0 ? 0.0 :
+      #endif
+      depthUnprojection[1] / (depth + depthUnprojection[0]);
   #else
-  originalDepth = depth;
+    #ifdef DEPTH_VISUALIZATER
+    highp float
+    #endif
+    originalDepth = depth;
+  #endif
+  #ifdef DEPTH_VISUALIZATER
+    highp float r = originalDepth / depthScaling;
+    highp float b = 0.5 - 0.5 * r;
+    fragmentColor = vec4(r, r, b, 1.0);
```

Comment:
I'd suggest to use colormaps for better visualization output -- [DebugTools::ColorMap](https://doc.magnum.graphics/magnum/namespaceMagnum_1_1DebugTools_1_1ColorMap.html). Especially the Turbo color map is tuned to make minor differences nicely visible and the output uniformly bright.

Further, to make it more generic, I'd name it a "color map transform" instead of "depth scaling", as that allows you to later use the same API for semantic ID visualization as well. [See how MeshVisualizer does it](https://doc.magnum.graphics/magnum/classMagnum_1_1Shaders_1_1MeshVisualizer3D.html#Shaders-MeshVisualizer-object-id).

Reply:
Cool stuff. I will study it.

Reply:
By the way, I understand you can apply such ColorMap in magnum built-in MeshVisusalizer shader. 
Question: how can I apply it to my customized shader??